docs/pull_hf_models.md

```diff
@@ -1,152 +1,36 @@
-```
-docker run -d --rm -v <model_repository_path>:/models openvino/model_server:latest \
---model_repository_path /models --list_models
+```bash
+docker run -d --rm -v <model_repository_path>:/models openvino/model_server:latest --pull --source_model <model_name_in_HF> --model_repository_path /models --model_name <external_model_name> --task <task> [TASK_SPECIFIC_PARAMETERS]
 ```
+:::
 
-For following directory structure:
-```
-/models
-├── meta
-│   ├── llama4
-│   │   └── graph.pbtxt
-│   ├── llama3.1
-│   │   └── graph.pbtxt
-├── LLama3.2
-│   └── graph.pbtxt
-└── resnet
-    └── 1
-        └── saved_model.pb
-```
+:::{tab-item} On Baremetal Host
+**Required:** OpenVINO Model Server package - see [deployment instructions](../deploying_server_baremetal.md) for details.
 
-The output would be:
-```
-meta/llama4
-meta/llama3.1
-LLama3.2
-resnet
+```bat
+ovms.exe --pull --source_model <model_name_in_HF> --model_repository_path <model_repository_path> --model_name <external_model_name> --task <task> [TASK_SPECIFIC_PARAMETERS]
 ```
+:::
+::::
 
-# Enable model
-
-To add model to ovms configuration file with specific model use either:
+Example for pulling `OpenVINO/Phi-3-mini-FastDraft-50M-int8-ov`:
 
+```bash
+docker run -d --rm -v /models:/models openvino/model_server:latest --source_model "OpenVINO/Phi-3-mini-FastDraft-50M-int8-ov" --model_repository_path /models --model_name Phi-3-mini-FastDraft-50M-int8-ov --task text_generation 
 ```
-docker run -d --rm -v <model_repository_path>:/models openvino/model_server:latest \
---model_repository_path /models/<model_path> --add_to_config <config_file_directory_path> --model_name <name>
-```
-
-When model is directly inside `/models`.
 
-Or
+It will prepare all needed configuration files to support LLMS with OVMS in the model repository. Check [parameters page](./parameters.md) for detailed descriptions of configuration options and parameter usage.
 
-```
-docker run -d --rm -v <model_repository_path>:/models openvino/model_server:latest \
---add_to_config <config_file_directory_path> --model_name <name> --model_path <model_path>
-```
-when there is no model_repository specified.
-
-## TIP: Use relative paths to make the config.json transferable in model_repository across ovms instances.
-For example:
-```
-cd model_repository_path
-ovms --add_to_config . --model_name OpenVINO/DeepSeek-R1-Distill-Qwen-1.5B-int4-ov --model_repository_path .
-```
-
-# Disable model
-
-If you want to remove model from configuration file you can do it either manually or use command:
-
-```
-docker run -d --rm -v <model_repository_path>:/models openvino/model_server:latest \
---remove_from_config <config_file_directory_path> --model_name <name>
-```
-
-FIXME TODO TBD
-- adjust existing documentation to link with this doc
-- task, task_params to be updated explained
\ No newline at end of file
+*Note:*
+When using ```--task``` you need both read and write access rights to models repository.
```

Comment:
```suggestion
When using ```--models_repositroy_path``` you need both read and write access rights to models repository.
```

Reply:
--models_repository can be used with list_models as well